<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="10" passed="8" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-08-10T17:11:47 IST" name="Default suite" finished-at="2025-08-10T17:13:07 IST" duration-ms="79984">
    <groups>
    </groups>
    <test started-at="2025-08-10T17:11:47 IST" name="Default test" finished-at="2025-08-10T17:13:07 IST" duration-ms="79984">
      <class name="TelecomProject.TelecomTest">
        <test-method signature="AddNewUser()[pri:1, instance:TelecomProject.TelecomTest@72d1ad2e]" started-at="2025-08-10T17:11:47 IST" name="AddNewUser" finished-at="2025-08-10T17:11:51 IST" duration-ms="3824" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddNewUser -->
        <test-method signature="getUserProfile()[pri:2, instance:TelecomProject.TelecomTest@72d1ad2e]" started-at="2025-08-10T17:11:51 IST" name="getUserProfile" finished-at="2025-08-10T17:11:53 IST" depends-on-methods="TelecomProject.TelecomTest.AddNewUser" duration-ms="1561" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getUserProfile -->
        <test-method signature="updateUser()[pri:2, instance:TelecomProject.TelecomTest@72d1ad2e]" started-at="2025-08-10T17:11:53 IST" name="updateUser" finished-at="2025-08-10T17:11:54 IST" depends-on-methods="TelecomProject.TelecomTest.AddNewUser" duration-ms="1838" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateUser -->
        <test-method signature="loginUser()[pri:2, instance:TelecomProject.TelecomTest@72d1ad2e]" started-at="2025-08-10T17:11:54 IST" name="loginUser" finished-at="2025-08-10T17:11:56 IST" depends-on-methods="TelecomProject.TelecomTest.updateUser" duration-ms="1680" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginUser -->
        <test-method signature="addContact()[pri:2, instance:TelecomProject.TelecomTest@72d1ad2e]" started-at="2025-08-10T17:11:56 IST" name="addContact" finished-at="2025-08-10T17:11:58 IST" depends-on-methods="TelecomProject.TelecomTest.loginUser" duration-ms="1696" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addContact -->
        <test-method signature="getContact()[pri:2, instance:TelecomProject.TelecomTest@72d1ad2e]" started-at="2025-08-10T17:11:58 IST" name="getContact" finished-at="2025-08-10T17:11:59 IST" depends-on-methods="TelecomProject.TelecomTest.loginUser" duration-ms="1739" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getContact -->
        <test-method signature="getContactList()[pri:2, instance:TelecomProject.TelecomTest@72d1ad2e]" started-at="2025-08-10T17:11:59 IST" name="getContactList" finished-at="2025-08-10T17:12:02 IST" depends-on-methods="TelecomProject.TelecomTest.loginUser" duration-ms="2044" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getContactList -->
        <test-method signature="logoutUser()[pri:2, instance:TelecomProject.TelecomTest@72d1ad2e]" started-at="2025-08-10T17:12:02 IST" name="logoutUser" finished-at="2025-08-10T17:12:03 IST" depends-on-methods="TelecomProject.TelecomTest.loginUser" duration-ms="1946" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutUser -->
        <test-method signature="updateContact()[pri:2, instance:TelecomProject.TelecomTest@72d1ad2e]" started-at="2025-08-10T17:12:03 IST" name="updateContact" finished-at="2025-08-10T17:12:35 IST" depends-on-methods="TelecomProject.TelecomTest.loginUser" duration-ms="31648" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [503]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [503]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:1418)
at org.testng.Assert.assertEquals(Assert.java:1382)
at org.testng.Assert.assertEquals(Assert.java:1428)
at TelecomProject.TelecomTest.updateContact(TelecomTest.java:151)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateContact -->
        <test-method signature="updateContacts()[pri:2, instance:TelecomProject.TelecomTest@72d1ad2e]" started-at="2025-08-10T17:12:35 IST" name="updateContacts" finished-at="2025-08-10T17:13:07 IST" depends-on-methods="TelecomProject.TelecomTest.loginUser" duration-ms="31938" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [503]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [503]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:1418)
at org.testng.Assert.assertEquals(Assert.java:1382)
at org.testng.Assert.assertEquals(Assert.java:1428)
at TelecomProject.TelecomTest.updateContacts(TelecomTest.java:134)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateContacts -->
      </class> <!-- TelecomProject.TelecomTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
